configs/sd-concepts.txt

```diff
@@ -0,0 +1,803 @@
+sd-concepts-library/001glitch-core
```

Comment:
this will fail with the \w regex

(this would actually be a good opportunity to make a unit test that pushes all these strings through your regex and only passes if they match; then you can make the regex as restrictive as possible for security reasons while ensuring it still does what it needs)